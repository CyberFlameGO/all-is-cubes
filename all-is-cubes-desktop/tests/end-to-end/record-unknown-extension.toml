# Tests the error reported if the output file doesn't have a known extension
bin.name = "all-is-cubes"
args = "--no-config-files --graphics record -o should-not-be-written.nonsense"
status.code = 2
