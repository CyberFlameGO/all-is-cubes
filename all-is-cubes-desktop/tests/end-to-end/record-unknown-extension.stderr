error: Invalid value 'should-not-be-written.nonsense' for '--output <FILE>': file name must have an extension specifying the type; one of 'png', 'apng', or 'gltf'

For more information try '--help'
